playbooks/archivematica-bionic/Vagrantfile

```diff
@@ -0,0 +1,49 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  config.vm.box = ENV.fetch("VAGRANT_BOX", "ubuntu/bionic64")
+
+  {
+    "am-local" => {
+      "ip" => "192.168.168.196",
```

Comment:
We've been allocating unique IPs for the different playbooks/****** environments, should the value here be different too?